src/main/java/seedu/partyplanet/commons/util/EditAutocompleteUtil.java

```diff
@@ -0,0 +1,91 @@
+package seedu.partyplanet.commons.util;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.partyplanet.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.partyplanet.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static seedu.partyplanet.logic.parser.CliSyntax.PREFIX_BIRTHDAY;
+import static seedu.partyplanet.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.partyplanet.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.partyplanet.logic.parser.CliSyntax.PREFIX_PHONE;
+import static seedu.partyplanet.logic.parser.CliSyntax.PREFIX_REMARK;
+import static seedu.partyplanet.logic.parser.CliSyntax.PREFIX_TAG;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javafx.collections.ObservableList;
+import seedu.partyplanet.commons.core.Messages;
+import seedu.partyplanet.commons.core.index.Index;
+import seedu.partyplanet.logic.commands.EditCommand;
+import seedu.partyplanet.logic.commands.HelpCommand;
+import seedu.partyplanet.logic.commands.exceptions.CommandException;
+import seedu.partyplanet.logic.parser.ArgumentMultimap;
+import seedu.partyplanet.logic.parser.ArgumentTokenizer;
+import seedu.partyplanet.logic.parser.ParserUtil;
+import seedu.partyplanet.logic.parser.exceptions.ParseException;
+import seedu.partyplanet.model.Model;
+import seedu.partyplanet.model.person.Person;
+
+public class EditAutocompleteUtil {
+
+    public static final String REMARK_PREFIX_NOT_PRESENT_MESSAGE = "Remark prefix not present!";
+    public static final String REMARK_NOT_EMPTY = "Remark not currently empty!";
```

Comment:
This might sound a little bit ambiguous, as the user may confuse it as the person he wants to edit does not have an empty remark. Maybe can change it to something like “You have already typed something!” ?

Reply:
Adopted suggestion!